packages/node/src/node.ts

```diff
@@ -1 +1,113 @@
-export default class Node {}
+import * as cf from "@counterfactual/cf.js";
+import { EventEmitter } from "events";
+import * as _ from "lodash";
+
+import { MultisigAddress, StateChannelInfo } from "./channel";
+
+export interface NodeOptions {
```

Comment:
is this necessary? why not just put it in the constructor and only abstract it as an interface if like 2 or more other classes end up using the same thing. seems like we're adding too many unnecessary things like this in the code